test/functional/tests/cli/test_cli_help_and_version.py

```diff
@@ -1,11 +1,12 @@
 #
-# Copyright(c) 2020-2021 Intel Corporation
+# Copyright(c) 2020-2022 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-import pytest
 import re
 
+import pytest
+
```

Comment:
Why this change? Most of the tests import pytest before other imports. Do we want to change this convention?

Reply:
I suspect this is result of "optimize imports", I don't recall to do such changes, there was no need. I can revert this if it can make any difference - does it?